plugins/scaffolder-react/src/next/components/ScaffolderWizardContextMenu/ScaffolderWizardContextMenu.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import IconButton from '@material-ui/core/IconButton';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Popover from '@material-ui/core/Popover';
+import { makeStyles } from '@material-ui/core/styles';
+import Edit from '@material-ui/icons/Edit';
+import MoreVert from '@material-ui/icons/MoreVert';
+import React, { useState } from 'react';
+
+const useStyles = makeStyles(theme => ({
+  button: {
+    color: theme.page.fontColor,
+  },
+}));
+
+/**
+ * @alpha
+ */
+export type ScaffolderWizardContextMenuProps = {
+  onEditorClicked?: () => void;
+};
+
+/**
+ * @alpha
```

Comment:
I don't think that we need to export this all the way out of the package. It can be kept a local export, so you can drop the `alpha` tags, and remove this component underneath the `TemplateWizardPage` folder instead?